app/org/sagebionetworks/bridge/dao/ParticipantOption.java

```diff
@@ -1,22 +1,74 @@
 package org.sagebionetworks.bridge.dao;
 
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import org.sagebionetworks.bridge.BridgeUtils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
 public enum ParticipantOption {
 
-    SHARING_SCOPE(SharingScope.NO_SHARING.name()),
-    EMAIL_NOTIFICATIONS(Boolean.TRUE.toString()),
-    EXTERNAL_IDENTIFIER(null),
-    DATA_GROUPS(null),
-    LANGUAGES(null);
+    SHARING_SCOPE(SharingScope.NO_SHARING.name(), "sharingScope") {
+        public String deserialize(JsonNode node) {
+            checkNotNull(node);
+            return SharingScope.valueOf(node.asText().toUpperCase()).name();
```

Comment:
This is user input. If the caller passes in an invalid sharing scope, Bridge server will throw a 500. We should add a try-catch here or in ParticipantController to wrap the exception as a BadRequest.